### PR TITLE
3.12 blogpost updates

### PR DESCRIPTION
**Pull request by christinaforney**
- Adding media
- Improved wording/clarification

### CODE REVIEW

File: blogposts/announcing-sourcegraph-3.12.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Sourcegraph 3.12: Match case toggle, draft campaigns, and a bunch of experimental features"
+title: "Sourcegraph 3.12: Match case toggle, draft campaigns, and some experimental features"
```
Comment (adaaaam) [inline blogposts/announcing-sourcegraph-3.12.md]:
```suggestion
title: "Sourcegraph 3.12: Match case toggle, draft campaigns, and exciting experimental features"
```

File: blogposts/announcing-sourcegraph-3.12.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ published: true
 
 Sourcegraph is the standard developer platform for code search and navigation at many of the largest and most advanced technology companies. With Sourcegraph, every company has access to the same kind of tools that Google and Facebook developers use every day.
 
-We're excited to announce Sourcegraph 3.12. This release contains a bunch of exciting experimental features we are looking forward to getting your feedback on!
+We're excited to announce Sourcegraph 3.12. This release contains some exciting experimental features we are looking forward to getting your feedback on!
```
Comment (adaaaam) [inline blogposts/announcing-sourcegraph-3.12.md]:
```suggestion
We're excited to announce Sourcegraph 3.12. This release contains exciting experimental features that we are looking forward to getting your feedback on!
```

File: blogposts/announcing-sourcegraph-3.12.md [hunk 3]
```diff
@@ -26,8 +26,10 @@ We're excited to announce Sourcegraph 3.12. This release contains a bunch of exc
 
 [**🛠 GitHub authentication now supports org membership restriction**](#github-authentication-now-supports-org-membership-restriction)<br />
 
+[**🔎 Search-based code intelligence indicator**](#search-based-code-intelligence-indicator)<br />
+
 [**🧪 Experimental features**](#experimental-features)<br />
-Interactive search mode, search across multiple revisions, out of the box repository permissions, faster permissions fetching on Bitbucket Server, and code intelligence precision indicator.
+Interactive search mode, search across multiple revisions, repository permissions API, faster permissions fetching on Bitbucket Server, and code intelligence precision indicator.
```
Comment (adaaaam) [inline blogposts/announcing-sourcegraph-3.12.md]:
```suggestion
Interactive search mode, search across multiple revisions, repository permissions API, faster permissions fetching on Bitbucket Server, and code intelligence precision indicator
```

File: blogposts/announcing-sourcegraph-3.12.md [hunk 4]
```diff
@@ -110,22 +118,16 @@ To enable this feature add `{ "experimentalFeatures": { "splitSearchModes": true
...
+Sourcegraph now provides a GraphQL API for setting repository permissions. Teams with complex permissioning requirements, such as having multiple code hosts, will benefit from setting up access controls directly in Sourcegraph. To enable Sourcegraph repository permissions, add `{ "permissions.userMapping" { "enabled": true, "bindID": "email" } }` to your site configuration. See the [explicit permissions API](https://docs.sourcegraph.com/admin/repo/permissions#explicit-permissions-api) documentation for more details.
 
 ### Faster permissions fetching on Bitbucket Server
 
 ![Faster permissions fetching chart](/images/3-12-faster-permissions.png "Faster permissions fetching chart")
 
 Previously, fetching ACL data from Bitbucket Server was limited by API constraints. Enhancements to the [Sourcegraph Bitbucket Server plugin](https://docs.sourcegraph.com/integration/bitbucket_server#sourcegraph-native-code-intelligence-plugin) have made fetching Bitbucket Server ACL data faster. To take advantage of this improvement install the Bitbucket Server plugin and add `{ "experimentalFeatures": { "bitbucketServerFastPerm": "enabled" } }` to your site configuration.
```
Comment (christinaforney) [inline blogposts/announcing-sourcegraph-3.12.md]: From @adaaaam 
```suggestion
Previously, fetching ACL data from Bitbucket Server was limited by API constraints. Enhancements to the [Sourcegraph Bitbucket Server plugin](https://docs.sourcegraph.com/integration/bitbucket_server#sourcegraph-native-code-intelligence-plugin) have made fetching Bitbucket Server ACL data faster. To take advantage of this improvement, install the Bitbucket Server plugin and add `{ "experimentalFeatures": { "bitbucketServerFastPerm": "enabled" } }` to your site configuration.
```

**Pull request merged**